vizro-ai/src/vizro_ai/_vizro_ai.py

```diff
@@ -19,41 +18,34 @@ class VizroAI:
     pipeline_manager: PipelineManager = PipelineManager()
```

Comment:
We should get rid of the `model_constructor` now, as I think it is not needed anymore.